lambdas/ecs-events-to-slack/notify.py

```diff
@@ -0,0 +1,142 @@
+import json
+import boto3
+import requests
```

Comment:
nit but these should be alphabetized!  (weirdly I'm surprised the pep8 linter you were using didn't yell at that!)